huggle/apiquery.cpp

```diff
@@ -416,6 +417,10 @@ void ApiQuery::SetAction(const Action action)
             this->ActionPart = "login";
             this->EnforceLogin = false;
             return;
+        case ClientLogin:
```

Comment:
This needs to be called ActionClientLogin

Reply:
done